CMakeLists.txt

```diff
@@ -140,6 +140,10 @@ if ( CYGWIN AND NOT CYGWIN_USE_POSIX )
 	message(STATUS "HAVE CYGWIN. Setting backward compat CMAKE_LEGACY_CYGWIN_WIN32 and -DWIN32")
 endif()
 
+if (DEFINED SRT_TEST_FAKE_LOSS)
+	add_definitions(-DSRT_TEST_FAKE_LOSS=${SRT_TEST_FAKE_LOSS})
```

Comment:
I think we should not expose this mode.

Reply:
We do anyway, just we don't need it exposed so ostentatious way. Any symbol that would be passed to the compiler as `-DSOME_FLAG=X` can be now enforced during `./configure` using `--enforce-some-flag=X` option (this has come in through some other PR). So it's now unnecessary.